website/src/styles/globals.css

```
@@ -0,0 +1,114 @@
+  --foreground-muted: #6e6e73;
+  --foreground-faint: #9a9aa0;
+  --background: #ffffff;
+  --surface: #ffffff;
+  --surface-soft: #f6f6f3;
+  --surface-warm: #fbfaf7;
+  --border: #e8e8e6;
+  --border-strong: #d6d6d2;
+  --accent: #d94e18;
+  --accent-hover: #b84315;
+  --accent-soft: #fdf0e8;
+  --font-display: "Fraunces", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)